tests/unit/migrator/test_migrator_api.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+from mock import MagicMock, patch
+
+from inspirehep.migrator.api import continuous_migration
+
+
+@patch("inspirehep.migrator.api.current_app")
+@patch("inspirehep.migrator.api.StrictRedis.from_url")
+@patch("inspirehep.migrator.api.Lock")
+@patch("inspirehep.migrator.api.insert_into_mirror")
+@patch("inspirehep.migrator.api.zlib")
+@patch("inspirehep.migrator.api.migrate_from_mirror")
+@patch("inspirehep.migrator.api.wait_for_all_tasks")
+def test_continous_migration_when_last_record_migrated_wont_be_added_to_mirror_properly(
```

Comment:
```suggestion
def test_continuous_migration_to_add_the_last_migrated_record_in_mirror(
```
?

Reply:
This name means something completely different...